app/screens/move-money-screen/move-money-screen.tsx

```diff
@@ -204,50 +207,72 @@ export const MoveMoneyScreenDataInjected: ScreenType = ({
     }
   }
 
-  const lastTransactions = find(data?.wallet, { id: "BTC" })?.transactions?.slice(
-    undefined,
-    3,
-  )
-
   return (
     <MoveMoneyScreen
       navigation={navigation}
       walletIsActive={walletIsActive(client)}
       loading={loadingMain}
       error={error}
-      amount={balanceUsd}
-      amountOtherCurrency={balanceBtc(client)}
       refetch={refetch}
       isUpdateAvailable={
         isUpdateAvailableOrRequired({ buildParameters: data?.buildParameters }).available
       }
-      transactions={lastTransactions}
+      hasToken={hasToken}
     />
   )
 }
 
+const RecentTransactions = ({ navigation }) => {
+  const { error, loading, data } = useQuery(TRANSACTIONS_LIST, {
+    variables: { first: 3 },
+    onError: console.error,
+  })
+
+  if (error) {
+    console.error(error)
+    toastShow("Error loading recent transactions.")
+    return null
+  }
+
+  if (loading) {
+    return <Text style={styles.transactionsLoading}>Loading...</Text>
+  }
+
+  const transactionsEdges = data.me.defaultAccount.wallets[0].transactions.edges
```

Comment:
Needs null checks on me to prevent a crash. Might as well add them to data and defaultAccount too.